frontend/src/worksheets/FiltersAndNewWorkSheet.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import FilterWorkSheetsByStatus from './FilterWorkSheetsByStatus'
+import FilterWorkSheetsByDate from './FilterWorkSheetsByDate'
```

Comment:
Kicsit redundás az elnevezés nekem, nem lenne olvashatóbb egy StatusFilter meg egy DateRangeFilter? Mivel a worksheets mappában vagyunk így látható, hogy ez a munkalapokhoz tartozó szűrő.